conversion/ena/submission.py

```diff
@@ -10,104 +12,64 @@
 from .experiment import EnaExperimentConverter
 from .run import EnaRunConverter
 
+CONVERSION_MAP = [
+    ('study', EnaProjectConverter()),
+    ('study', EnaStudyConverter()),
```

Comment:
It works to submit both and allows us to customise the project and study separately.
When we split up the project and separate what is config for covid and what's supported for all users it would be useful to have a working model for both `study` and `project` even if each user would only use either `study` or `project`.

Reply:
Well, both the Covid team and Rasko said that only need to submit one of them as the other is getting created at ENA side. I would rather do only one of them as there is also less opportunity for errors.